test/components/views/rooms/__snapshots__/RoomPreviewBar-test.tsx.snap

```diff
@@ -116,10 +116,40 @@ exports[`<RoomPreviewBar /> with an invite with an invited email when client has
   class="mx_RoomPreviewBar_message"
 >
   <h3>
-    This invite to RoomPreviewBar-test-room was sent to test@test.com
+    Do you want to join RoomPreviewBar-test-room?
```

Comment:
Are we expecting this change in the snapshot? It looks like what it prints does not match the test name anymore, could we update it?

Reply:
Yes. It does actually match, it was wrong before. The test name is `renders invite message when invite email mxid match` yet it was the same snapshot as `renders email mismatch message when invite email mxid *doesnt* match`